### PR TITLE
Dg update

### PR DESCRIPTION
**Pull request by underthehai**
Add test cases to manual testing portion
Update edit sequence diagram

### CODE REVIEW

**Review by lyeyixian — APPROVED** [review]
LGTM

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -881,12 +881,106 @@ testers are expected to do more *exploratory* testing.
...
    1. Re-launch the app by double-clicking the jar file.<br>
        Expected: The most recent window size and location is retained.
+       
+### Deleting
 
+1. Deleting a friend while all friends are being shown
+
+   1. Prerequisites: Show all friends using the `show -friend` command. Multiple Friends in the list.
+
+   1. Test case: `delete -friend 1`<br>
```
Comment (jiaweiteo) [inline docs/DeveloperGuide.md:899]: Should all of these be listed as 2., 3., ... onwards?

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -881,12 +881,106 @@ testers are expected to do more *exploratory* testing.
...
+   1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
+      Expected: Similar to previous.
+
+### Adding
+
+1. Adding a friend while in a travel plan directory.
+
+   1. Prerequisites: Navigate to a valid directory using `goto -travelplan x` (where x is a valid index)
+   
+   1. Test case: `add -friend n/Tom p/E1234567X m/81234567`<br>
```
Comment (jiaweiteo) [inline docs/DeveloperGuide.md:914]: Same as above.

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -881,12 +881,106 @@ testers are expected to do more *exploratory* testing.
...
+   1. Other incorrect add commands to try: `edit -activity`<br>
+      Expected: Error message is thrown.
+
+### Show
+
+1. Show Accommodation while in a travel plan directory.
+
+   1. Prerequisites: Currently in a travel plan directory
+   
+   1. Test case: `show -accommodation x`<br>
```
Comment (jiaweiteo) [inline docs/DeveloperGuide.md:949]: there is no `x` after `show -accommodation`

**Review by jiaweiteo — APPROVED** [review]
LGTM, just need to change some typos and listing format

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -881,12 +881,106 @@ testers are expected to do more *exploratory* testing.
 
    1. Download the jar file and copy into an empty folder
 
-   1. Double-click the jar file Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
+   1. Run jar file in command prompt: Expected: Shows the GUI with a set of sample data. The window size may not be optimum.
```
Comment (jiaweiteo) [inline docs/DeveloperGuide.md:884]: Can change to `Run jar file in command prompt and enter 'java -jar wanderlust.jar'`.

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -881,12 +881,106 @@ testers are expected to do more *exploratory* testing.
...
 
 1. Saving window preferences
 
    1. Resize the window to an optimum size. Move the window to a different location. Close the window.
 
    1. Re-launch the app by double-clicking the jar file.<br>
```
Comment (jiaweiteo) [inline docs/DeveloperGuide.md:890]: Can change to `Re-launch the app by running on the command prompt and enter  'java -jar wanderlust.jar`.

**Review by jiaweiteo — APPROVED** [review]

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (underthehai) [reply docs/DeveloperGuide.md:899]: it will turn out in order as i ii iii ...

**Pull request merged**